src/ex5/dist/clients/item_client.js

```diff
@@ -1,9 +1,10 @@
 const url = `http://localhost:8080/api`;
+const header = { "Content-Type": "application/json" };
```

Comment:
```suggestion
const HEADERS = { "Content-Type": "application/json" };
```
const name should be with upper case latter